### PR TITLE
Remove job from resize&execResize

### PR DESCRIPTION
**Pull request by runcom**
#12151

Signed-off-by: Antonio Murdaca me@runcom.ninja

### CODE REVIEW

File: api/server/server.go [hunk 1]
```diff
@@ -985,9 +985,28 @@ func postContainersResize(eng *engine.Engine, version version.Version, w http.Re
...
+	cont, err := d.Get(vars["name"])
+	if err != nil {
+		return err
+	}
+
+	if err := cont.Resize(height, width); err != nil {
 		return err
 	}
+
+	// why no header is even set or http 204?
```
Comment (runcom) [inline api/server/server.go]: suggestions?

File: daemon/resize.go [hunk 2]
```diff
@@ -1,48 +1,6 @@
...
-	name := job.Args[0]
-	height, err := strconv.Atoi(job.Args[1])
-	if err != nil {
-		return err
-	}
-	width, err := strconv.Atoi(job.Args[2])
-	if err != nil {
-		return err
-	}
+func (daemon *Daemon) ContainerExecResize(name string, height, width int) error {
```
Comment (runcom) [inline daemon/resize.go]: not sure if exporting `getExecConfig` from daemon would have been a good idea so I just left this here in resize.go

File: api/server/server.go (hunk 1 above)
Comment (LK4D4) [inline api/server/server.go]: I think `http` writes 200 automatically on handler return.

Comment (runcom) [inline api/server/server.go]: gotcha

File: daemon/resize.go (hunk 2 above)
Comment (LK4D4) [inline daemon/resize.go]: Yeah, exec is tough. I think it is okay for now.

File: api/server/server.go (hunk 1 above)
Comment (LK4D4) [inline api/server/server.go]: Wanna remove your comments?

**Comment by LK4D4** [comment]
@runcom Need rebase

File: api/server/server.go (hunk 1 above)
Comment (runcom) [inline api/server/server.go]: Yes, back at my laptop in some minutes and I'll push force

**Comment by runcom** [comment]
@LK4D4 rebased and removed comments

**Comment by LK4D4** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**